requirements.in

```diff
@@ -1,4 +1,5 @@
 beautifulsoup4~=4.11.1
+blessed~=1.20.0
```

Comment:
Wondering about this addition here, what is it for?

Reply:
It's a terminal UI library. It was previously included innately in the Q1, but now it's required explicitly for the `python manage.py qinfo` command to work.

Reply:
The earlier dependency from Q1 is visible in this line of the change: https://github.com/City-of-Helsinki/mvj/pull/717/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552L39

Reply:
That is a bit annoying, now it is harder to figure out why we have this blessed dependency when we don't directly use it. q2 should require its own dependencies it needs. :/
Looks like it was removed here https://github.com/django-q2/django-q2/pull/183
But it doesn't mention why. But then again it was supposedly removed earlier, but not from the requirements file: https://github.com/django-q2/django-q2/pull/20

I am wondering could this be a dev dependency instead, and if it is needed in production it could be installed separately? Would be better not to have a dependency that is rarely used.

Reply:
Makes sense, I'll set it as a dev dependency.